exporter/elasticsearchexporter/README.md

```diff
@@ -129,10 +129,6 @@ behaviours, which may be configured through the following settings:
     - `raw`: Omit the `Attributes.` string prefixed to field names for log and 
              span attributes as well as omit the `Events.` string prefixed to
              field names for span events. 
-  - `dedup` (default=true; DEPRECATED, in future deduplication will always be enabled):
```

Comment:
Let's not remove the option from the docs. Let's update the description to say it doesn't do anything.

Reply:
Will do.